app/views/rubygems/transfer/_summary.html.erb

```diff
@@ -20,14 +20,32 @@
 </div>
 <hr class="border-neutral-400 my-6">
 
+<div class="mb-6">
+  <div class="flex justify-between items-center">
+    <h3 class="text-b1 text-neutral-900 dark:text-white">
+      Gems
+      <span class="text-neutral-600 font-light ml-1"><%= @rubygem_transfer.selected_rubygems.size %></span>
+    </h3>
+    <% if current_step > 2 %>
```

Comment:
🎏 Noting that this would be easy **for me** to miss if I was making a change which involved steps being reordered. That doesn't mean it's invalid, or the wrong thing to do here.